### PR TITLE
*: upgarde all docker images to use rocky linux 8

### LINKED ISSUE #9586: OALP perftest  tpch-2tiflash-50g& tpch-tiflash-50g result with total-avg-latency raise 15-20%
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
Deploy a tidb cluster with 3 TiDB (16c,48G) + 3 TiKV (16c,48G) + 2 TiFlash (16c,80G)
Compare the performance of v8.4.0 and v8.3.0 with TPCH benchmark

### 2. What did you expect to see? (Required)
No performance regression

### 3. What did you see instead (Required)
total-avg-latency raise 15-20%.
image

### 4. What is your TiFlash version? (Required)
v8.4.0 (cea728d5d2bad7766dfc5e97cdf8ef16fe4a791f)

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### What problem does this PR solve?

Issue Number: close #9586

Problem Summary:

### What is changed and how it works?

```commit-message
*: upgarde all docker images to use rocky linux 8
```

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by wuhuizuo** [comment]
/hold

It will affect the build instructors used in CI/CD flows.

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ sudo pacman -S clang lld libc++ libc++abi compiler-rt openmp lcov cmake ninja cu
 <details>
 <summary><b>CentOS 7</b></summary>
```
Comment (JaySon-Huang) [inline README.md:107]:
```suggestion
<summary><b>Rocky Linux 8</b></summary>
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Pull request merged**